vuepress/docs/v6.3.2/tutorials/samples/landing-page.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Solution Templates 
+
+The Entando Application Composition Platform offers several Solution Templates to demonstrate various business capabilities and integrations:
+
+- **Entando Customer Portal**
+- [**Entando Hub**](hub.md)
+- [**Entando Process Driven Applications Plugin**](pda-tutorial.md)
+- [**Entando Standard Banking Demo**](install-standard-demo.md)
+
+Each Open Source Solution Template was built with Entando and can be used as-is, reviewed for common development practices, or leveraged as a useful starting point for a related application. 
```

Comment:
Let's lowercase Open Source. Seems off somehow unless I missed a discussion on that.